contrib/log_correlation/stackdriver/README.md

```diff
@@ -1,10 +1,141 @@
 # OpenCensus Stackdriver Log Correlation
 
-This subproject is currently experimental. It provides a Stackdriver Logging
-[LoggingEnhancer](http://googlecloudplatform.github.io/google-cloud-java/google-cloud-clients/apidocs/com/google/cloud/logging/LoggingEnhancer.html)
-that automatically adds tracing data to log entries. The LoggingEnhancer adds the trace ID, which
-allows Stackdriver to display log entries associated with each trace or filter logs based on trace
-ID. It currently also adds the span ID and sampling decision.
+This subproject is currently experimental, so it doesn't have any stability guarantees. It will
+remain experimental until we have a specification for a log correlation feature in
+[opencensus-specs](https://github.com/census-instrumentation/opencensus-specs/)
+(issue [#123](https://github.com/census-instrumentation/opencensus-specs/issues/123)).
+
+The `opencensus-contrib-log-correlation-stackdriver` artifact provides a
+[Stackdriver Logging](https://cloud.google.com/logging/)
+[`LoggingEnhancer`](http://googlecloudplatform.github.io/google-cloud-java/google-cloud-clients/apidocs/com/google/cloud/logging/LoggingEnhancer.html)
+that automatically adds tracing data to log entries. The class name is
+`OpenCensusTraceLoggingEnhancer`. `OpenCensusTraceLoggingEnhancer` adds the current trace and span
+ID to log entries, which allows Stackdriver to display the log entries associated with each trace,
+or filter logs based on trace or span ID. It currently also adds the sampling decision using the
+label "`openCensusTraceSampled`".
 
 TODO(sebright): Add a demo to https://github.com/census-ecosystem/opencensus-experiments and link to
 it.
+
+## Instructions
+
+### Prerequisites
+
+This log correlation feature requires a project that is using the
+[`com.google.cloud:google-cloud-logging`](https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-clients/google-cloud-logging)
+library to export logs to Stackdriver. `google-cloud-logging` must be version `1.33.0` or later.
+The application can run on Google Cloud Platform, on-premise, or on
+another cloud platform. See https://cloud.google.com/logging/docs/setup/java for instructions for
+setting up `google-cloud-logging`. Note that this artifact does not support logging done through the
+Stackdriver Logging agent.
+
```

Comment:
It might be better to add the Note on separate paragraph and perhaps in bold letters.